### ISSUE TITLE
Weather module: add feature "showOutdoorTemperature"

### ISSUE THREAD

**Issue by Coernel82** [MichMich/MagicMirror#3126]
The module already has the option to display the notification of ``INDOOR_TEMPERATURE`` by other modules. People also have outdoor sensors. So adding the functioncality for ``showOutdoorTemperature`` (assuming the notification ``OUTDOOR_TEMPERATURE`` is already existant.
This could then optionally replace or complement the outdoor temperature received by the weather provider.

After this https://github.com/splattner/MMM-MQTT-Notification could be used to send the notificaton if using MQTT.

**Comment by rejas** [comment]
Sounds like a useful feature indeed. Do you want to work on it? or would you want help with that?

**Comment by Coernel82** [comment]
I am a total noob. I can program some minimalistic PHP and CPP for Arduino. As much as I would love to this is far too complex for me I guess.
However I will have a look at the code and report back if I might be able to do it!

**Comment by Coernel82** [comment]
I had the promised look at the code and it seems as this could be established without to much hassle as the code already exists for the other broadcast variable.
For some reasons however I decided not to proceed this further

- first of all I am really a noob so even though the task appears to be easy I might make mistakes and spend a long time correcting them. This time I currently lack of
- after more research I found out there is already a MMM MQTT weather module available getting a ton of MQTT readings such as wind, moisture, lightlevel and much more. Maybe there should be some cooperation? Have a look at https://github.com/spitzlbergerj/MMM-Currentweather-MQTT


Thanks for your understanding!
